orga/app/controllers/authenticated/campaigns/list.js

```diff
@@ -21,11 +21,10 @@ export default class ListController extends Controller {
 
   @service currentUser;
 
-  @empty('model') hasNoCampaign;
-
   @computed('model')
   get displayNoCampaignPanel() {
-    return this.hasNoCampaign && isEmpty(this.name) && isEmpty(this.status) && isEmpty(this.creatorId);
+    const { hasCampaigns } = this.model.meta;
+    return !hasCampaigns && isEmpty(this.name) && isEmpty(this.status) && isEmpty(this.creatorId);
```

Comment:
du coup je pense que on peut supprimer les autres conditions, puisque à elle seule `hasCampaign` a l'état.
```suggestion
    return !hasCampaigns;
```